doc/rados/troubleshooting/log-and-debug.rst

```diff
@@ -138,6 +138,15 @@ Valgrind is computationally expensive, and will slow down your system otherwise.
 Valgrind messages are logged to ``stderr``. 
 
 
+ceph-fuse
+=========
+
+For ceph-fuse, try running in the foreground with logging to the
+console (``-d``) and enabling client debug (``--debug-client=20``), enabling
+prints for each message sent (``--debug-ms=1``) and debugging for the
+monitor client (``--debug-monc=20``).
```

Comment:
I wouldn't generally advise monc=20 for client debugging, it was just in that particular case that it seemed like a potential mon issue.